src/utils/ConfigUtils.js

```diff
@@ -147,6 +147,83 @@ const isInstanceViewConfigValid = (instanceView) => {
   return true;
 };
 
+const checkUnknownKeysInConfig = (schema, data) => {
+  try {
+    schema.parse(data);
+  } catch (e) {
+    e.issues
+      .filter((issue) => issue.code === 'unrecognized_keys')
+      .forEach((issue) => {
+        if (issue.path?.length === 0) {
+          if (data?.id?.toLowerCase().startsWith('w')) {
+            console.warn(
+              `Your workspace configuration contains unknown keys: '${issue.keys.join(
+                ', '
+              )}', please, check your configuration`
```

Comment:
Unnecessary comma after "please" ?
Same thing below, on line 166

Reply:
you are right, I used it without comma elsewhere, I fix it :slightly_smiling_face: